### PR TITLE
Feature/background color

### PR DESCRIPTION
**Pull request by nicandris**
- Add ability to enable/disable the background fog
- If fog is disabled then default background color is set based on your theme.
- Background color can be customized using a hex value eg. #3c3c3c if fog is disabled

image
image

### CODE REVIEW

**Comment by AntonyLeons** [comment]
Thanks for your contribution, i think we need to add to the setup flow too, not just environment variables, I can work on this in a few weeks, if you dont have time.

**Comment by nicandris** [comment]
I added it in the setup also. please take a good look since I'm rusty on js/html
image
image
image

**Comment by AntonyLeons** [comment]
the setup looks good, one issue though:
default behaviour has now changed fog is now disabled by default, consider calling the flag disable fog, at least internally.
this will change behaviour for people who auto update with docker.

**Comment by nicandris** [comment]
I added it as a "migration" method in IndexService. Can be moved to a separate migration service if needed

File: src/main/resources/templates/setup.html [hunk 1]
```diff
@@ -66,6 +66,8 @@
                             <input id = "light-theme" class = "first" type = "button" value = "LIGHT THEME" />
                             <input id = "dark-theme" class = "second" type = "button" value = "DARK THEME" />
                         </div>
+                        <label class = "label-additional-settings"><span>ENABLE BACKGROUND FOG</span><input id = "fog-toggle" class = "fog-toggle" type = "checkbox" name = "enableFog" checked></label>
+                        <label class = "label-additional-settings"><span>SELECT BACKGROUND COLOR</span><input id = "color-selector" class = "color-selector" type = "color" value="#e5e5e5" disabled></label>
```
Comment (AntonyLeons) [inline src/main/resources/templates/setup.html:70]:
```suggestion
                        <label class = "label-additional-settings" title="Disable Fog to select"><span>SELECT BACKGROUND COLOR</span><input id = "color-selector" class = "color-selector" type = "color" value="#e5e5e5" disabled></label>
```

**Comment by AntonyLeons** [comment]
looks good will merge soon, just a minor suggestion but doesn't matter.

**Pull request merged**